pkgs/tools/misc/colorz/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "colorz";
+  version = "1.0.3";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0ghd90lgplf051fs5n5bb42zffd3fqpgzkbv6bhjw7r8jqwgcky0";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ pillow scipy ];
+
```

be nice to have some sanity check that pillow and scipy are imported correctly
```suggestion
  checkPhase = ''
    $out/bin/colorz --help > /dev/null
  '';
```